fence/blueprints/admin.py

```diff
@@ -495,6 +495,19 @@ def add_policy_to_user():
 
     return jsonify(res)
 
+@blueprint.route("/list_policies", methods=["GET"])
+@admin_login_required
+@debug_log
+def list_policies():
+    """
+    Return a list of all policies. Returns in JSON format
+    """
+    expand = request.args.get('expand', default = False, type = bool)
```

Comment:
You should add a check that makes sure the value passed is a boolean rather than use the type parameter, as it will attempt to convert the input to a boolean and throw a type error if not possible, a UserError plus a message explaining what the input should be would be easier for the client to understand in the future as well as the numbers 1 and 0 would be allowed threw as valid parameters

Reply:
Actually yea dont use the type parameter to check cause almost everything can be cast as a bool in python